packages/mc-scripts/package.json

```diff
@@ -65,17 +65,17 @@
     "postcss-preset-env": "6.7.0",
     "postcss-reporter": "6.0.1",
     "postcss-safe-parser": "4.0.1",
-    "react-dev-utils": "9.0.1",
+    "react-dev-utils": "9.0.2",
     "shelljs": "0.8.3",
-    "style-loader": "0.23.1",
+    "style-loader": "1.0.0",
```

Comment:
<img width="917" alt="CleanShot 2019-08-12 at 09 30 37@2x" src="https://user-images.githubusercontent.com/1877073/62851113-5a4c6700-bce5-11e9-808f-e4661b6220b2.png">

I can't see us using any of these.